pkg/apis/kops/instancegroup.go

```diff
@@ -178,6 +181,16 @@ const (
 // SpotAllocationStrategies is a collection of supported strategies
 var SpotAllocationStrategies = []string{SpotAllocationStrategyLowestPrices, SpotAllocationStrategyDiversified, SpotAllocationStrategyCapacityOptimized}
 
+// InstanceMetadataOptions holds the fields required to configure instance metadata in an instance.
+// This currently holds fields to require EC2 Instance Metadata in EC2
+type InstanceMetadataOptions struct {
+	// HTTPPutResponseHopLimit is the max no of hops made by the PUT request which fetches the token(default '1')
```

Comment:
From: https://docs.aws.amazon.com/sdk-for-go/api/service/autoscaling/#InstanceMetadataOptions
```suggestion
	// HTTPPutResponseHopLimit is the desired HTTP PUT response hop limit for instance metadata requests.
	// The larger the number, the further instance metadata requests can travel. The default value is 1.
```